MODULE.bazel

```diff
@@ -0,0 +1,17 @@
+module(
+    name = "grpc-web",
+    version = "1.6.0.rc1",
```

Comment:
is the `.rc1` suffix required? Since we don't tend to do "RC"s, i'd think `1.6.0` should be able to uniquely identify the current version?

Reply:
Done.

Reply:
Thanks!!

Actually.. it should probably be 1.5.0 also? 😄